tests/ci_build/pip_tests/Dockerfile.in.pip_cpu

```diff
@@ -1,4 +1,4 @@
 # -*- mode: dockerfile -*-
 # dockerfile to test pip installation on CPU
 
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
why?

Reply:
Fixes problems with outdated versions of openblas which cause bugs and hangs, amongst others.

Reply:
OpenBlas on ubuntu14.04 is not compatible to the latest CPUs running in C5 instances. This could be fixed if OpenBlas released a new version on Ubuntu14.04, but so far they didn't and I wanted to refrain from downloading and compiling the latest version myself.